plugin-core/src/main/java/appland/cli/DefaultCommandLineService.java

```diff
@@ -476,18 +476,23 @@ private static boolean isSupported() {
                 || SystemInfo.isWindows && CpuArch.isIntel64();
     }
 
-    private static @NotNull KillableProcessHandler startProcess(@NotNull Path workingDir,
+    public static @NotNull KillableProcessHandler startProcess(@NotNull Path workingDir,
                                                                 @NotNull String... commandLine) throws ExecutionException {
 
+        final var settings = AppMapApplicationSettingsService.getInstance();
```

Comment:
I usually don't make local variables `final` to improve readability, but it's a matter of taste. I'm fine either way.

Reply:
Fair enough, I suppose it's a habit from JavaScript. I'll leave it as is.